dataplane/driver.go

```diff
@@ -297,6 +298,8 @@ func StartDataplaneDriver(configParams *config.Config,
 				InterfaceName:       configParams.WireguardInterfaceName,
 				MTU:                 configParams.WireguardMTU,
 				RouteSource:         configParams.RouteSource,
+				// currently, AKS *must* encrypt hostnetwork traffic, while it is optional on EKS
```

Comment:
This comment line probably should be moved inside `ShouldEncryptWireguardHostTraffic()`